UPGRADE-6.0.md

```diff
@@ -1,6 +1,11 @@
 UPGRADE FROM 5.x to 6.0
 =======================
 
+Console
+-------
+
+ * `Command::setHidden()` has a default value (`true`) for `$hidden` parameter
```

Comment:
This should be somewhere in the code. If not, we will forget to add the default value in 6.0.

Reply:
I added a note in the PHP Doc + `/*= true*/` in the method signature.